src/main/java/jenkins/plugins/slack/decisions/OnRepeatedFailure.java

```diff
@@ -0,0 +1,31 @@
+package jenkins.plugins.slack.decisions;
+
+import hudson.model.Result;
+import jenkins.plugins.slack.SlackNotifier;
+import jenkins.plugins.slack.logging.BuildAwareLogger;
+
+public class OnRepeatedFailure implements Condition {
```

Comment:
Gah, there should be an OnEveryFailure one imo, this behaviour is just confusing
At least I can now tell what's going on with this code

Reply:
Captured this as https://github.com/jenkinsci/slack-plugin/issues/520